src/App.js

```diff
@@ -2,12 +2,14 @@
 import './App.css';
 import Navbar from './components/HeaderComponents/Navbar'
```

Comment:
First external imports occur that is imports from installed libraries, then 1 line space, then all internal imports that is files/components/functions that you have created, then 1 line space and then finally all css imports.

Reply:
done sir